src/components/Keycode.vue

```diff
@@ -25,16 +25,47 @@ import isUndefined from 'lodash/isUndefined';
 
 const debug = false;
 export default {
-  name: 'keycode',
+  name: 'KeycodeComponent',
   props: {
-    type: String,
-    code: String,
-    title: String,
-    width: null,
-    name: String,
-    classes: String,
-    styles: Object,
-    layer: Number
+    type: {
+      type: String,
+      default: undefined
+    },
+    code: {
+      type: String,
+      default: ''
+    },
+    title: {
+      type: String,
+      default: ''
+    },
+    width: {
+      type: String,
+      default: undefined
+    },
```

Comment:
This was `width: null` previously - is that no longer valid?

Reply:
on L75 we check if it's undefined, so this is equivalent.